docs/account-management/api-keys/create-api-keys.md

```diff
@@ -117,7 +117,7 @@ else:
 </TabItem>
 <TabItem value="RA" label="REST API">
 
-Use our interactive API Reference with code generation in 18 programming languages to [Create an API Key](https://macrometa.com/docs/api#/operations/CreateApiKey).
+Use our interactive API Reference with code generation in 18 programming languages to [Create an API Key](https://www.macrometa.com/docs/api#/operations/CreateApiKey).
```

Comment:
@brandonLyonMM wondering if we should just make these relative links.. wdyt?

Reply:
That works for me. I didn't know if there was a reason they weren't relative.

Reply:
Because they are linking to the API reference, which I did not think we could make relative?